homeassistant/components/todo/services.yaml

```diff
@@ -55,7 +55,7 @@ update_item:
   fields:
     item:
       required: true
-      example: "Submit income tax return"
+      example: "62df5ffa78e84f90a0f7ff38aa28ff9d"
```

Comment:
while the UID is the best options in ensuring that a unique item is checked, it is also more complex, as it requires fetching it first. Personally, I think we should keep the item name in the example since that provides the simplest example.
Also, there are use cases where items are added and updated by automations (like adding "fold clothes" when the drying machine finishes, and having buttons around to automatically check items). For those use cases, the name is enough as it would be fixed.

Reply:
I agree that's what I wrote in the description of the PR or within the `strings.json`. I wanted to highlight the issue that can happen if we are not using the UID. 

About the yaml here, I didn't know how to approach it since but I can revert it.